app/containers/ItunesAppContainer/ErrorMessage/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * ErrorMessage
+ *
+ */
+
+import React from 'react';
+// import PropTypes from 'prop-types'
+// import styled from 'styled-components'
+import { Result, Button } from 'antd';
+
+import { FormattedMessage as T, injectIntl } from 'react-intl';
```

Comment:
we can use the T component, internally it uses FormattedMessage but provides some enhancements on top of it as well